frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -42,7 +42,22 @@
           :isRequired="input.is_required"
           :tooltipMessage="input.description"
           :visibleOptions="input.settings.visible_options"
-          @on-error="onError"
+          :optionsThresholdToEnableSeach="
```

Comment:
Maybe this property is not necessary. You can keep this logic internally for the Single/MultiLabel components. We can expose it later if we find a question settings attribute to enable/disable the search bar on labels.